test/new-e2e/examples/customenv_with_two_vm_test.go

```diff
@@ -26,6 +26,9 @@ type multiVMEnv struct {
 func multiVMEnvProvisioner() e2e.PulumiEnvRunFunc[multiVMEnv] {
 	return func(ctx *pulumi.Context, env *multiVMEnv) error {
 		awsEnv, err := aws.NewEnvironment(ctx)
+		if err != nil {
```

Comment:
I had a linter error for that locally so I fixed it, didn't realize it wasn't in the same file 😄

Reply:
Looks like the `examples` package is implicitly skipped by `golangci-lint`, I'll try to fix that